### PR TITLE
go to login page after jwt is expired

### PR DESCRIPTION
**Pull request by OfekN11**

### CODE REVIEW

**Comment by RoiTief** [comment]
Did you test this?
After 31 minutes I'm hit with these messages:
image

But I can still roam free in the system, I should be rerouted to the login page

**Comment by OfekN11** [comment]
WTF I tested it a lo of time

**Comment by OfekN11** [comment]
Ok my bad, I didnt expect different messages.
Fixed

File: src/main/package.json [hunk 1]
```diff
@@ -70,7 +71,7 @@
   "scripts": {
     "start": "node main.js 2>&1 | pino-pretty",
     "test": "jest",
-    "dev": "node main.js 2>&1 | pino-pretty"
+    "dev": "nodemon  main.js 2>&1 | pino-pretty"
```
Comment (RoiTief) [inline src/main/package.json:74]: Why suddenly do we need to change the way we run the server??😰😰

**Comment by RoiTief** [comment]
image

now crashes at compilation...

**Review by RoiTief — APPROVED** [review]

File: src/main/package.json (hunk 1 above)
Comment (OfekN11) [reply src/main/package.json:74]: When you save a change, it restart it.

**Pull request merged**